api-admin/src/main/java/bio/terra/pearl/api/admin/service/PopulateExtService.java

```diff
@@ -114,54 +128,65 @@ public SiteContent populateSiteContent(
     }
   }
 
+  @EnforcePortalStudyEnvPermission(permission = "BASE")
   public Enrollee populateEnrollee(
-      String portalShortcode,
-      EnvironmentName envName,
-      String studyShortcode,
-      String filePathName,
-      AdminUser user,
-      boolean overwrite) {
-    authorizeUser(user);
+      PortalStudyEnvAuthContext authContext, String filePathName, boolean overwrite) {
     StudyPopulateContext config =
         new StudyPopulateContext(
-            filePathName, portalShortcode, studyShortcode, envName, new HashMap<>(), false, null);
+            filePathName,
+            authContext.getPortalShortcode(),
+            authContext.getStudyShortcode(),
+            authContext.getEnvironmentName(),
+            new HashMap<>(),
+            false,
+            null);
     return enrolleePopulator.populate(config, overwrite);
   }
 
+  @EnforcePortalStudyEnvPermission(permission = "BASE")
+  public Enrollee populateEnrollee(
```

Comment:
Should we add an extra check here to prevent this endpoint from being used in `live`? I agree that this should be usable in Sandbox and IRB (as you've done in the frontend code)

Reply:
good question.  I waffled on that, since I could see a case where someone wants to create a synthetic participant in live to test some published configs (and then will withdraw them).  And so it's disabled on the frontend but not the backend :P   I'll disable it here too